components/sidebar_right/sidebar_right.jsx

```diff
@@ -73,12 +73,21 @@ export default class SidebarRight extends React.PureComponent {
     }
 
     handleShortcut = (e) => {
-        if (Utils.cmdOrCtrlPressed(e) && Utils.isKeyPressed(e, Constants.KeyCodes.PERIOD)) {
-            e.preventDefault();
-            if (this.props.isOpen) {
-                this.props.actions.closeRightHandSide();
-            } else {
-                this.props.actions.openAtPrevious(this.previous);
+        if (Utils.cmdOrCtrlPressed(e)) {
+            if (e.shiftKey && Utils.isKeyPressed(e, Constants.KeyCodes.PERIOD)) {
+                e.preventDefault();
+                if (this.props.isOpen) {
+                    this.props.actions.setRhsExpanded(true);
+                } else {
+                    this.props.actions.openAtPrevious(this.previous);
+                }
+            } else if (Utils.isKeyPressed(e, Constants.KeyCodes.PERIOD)) {
+                e.preventDefault();
+                if (this.props.isOpen) {
+                    this.props.actions.closeRightHandSide();
+                } else {
+                    this.props.actions.openAtPrevious(this.previous);
+                }
```

Comment:
I tested the PR by running it locally i have observed the following scenarios
- `CMD SHIFT .` on no RHS, opens the RHS (✔️ )
- `CMD SHIFT .` on opened RHS, expands THS (✔️ )
- `CMD SHIFT .` on expanded RHS, does nothing currently, this behavior was not recorded on jira too. what should be expended on this scenarios @esethna

Reply:
@M-ZubairAhmed good question. I would imagine that `CMD Shift .` when the RHS is already expanded should shrink the RHS back to normal width

Feel free to add to the JIRA. cc// @codedsun

Reply:
Awesome thank you @codedsun  can you add the above scenario too?